cmd/manager/main.go

```diff
@@ -95,6 +100,25 @@ func main() {
 	log.Info("Registering 3rd party resources")
 	registerAdditionalResources(mgr)
 
+	// load plugins
+	currentDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	pluginsDir := filepath.Join(currentDir, "plugins")
```

Comment:
Is it documented somewhere what should be done to add a plugin, where, how to build it , ... ?

Reply:
cf this PR's description, there is no documentation at the moment.